ross/rotor_assembly.py

```diff
@@ -1389,6 +1391,28 @@ def load(file_name):
         with open(file_name, "rb") as f:
             return pickle.load(f)
 
+    def static(self):
+        """This function calculates the static displacement of a rotor
+        due the disks' weight, and returns an array with the nodal displacement
+        The calculation is done from the stiffness and mass matrix of a rotor.
+        """
+        # gravity aceleration vector
+        grav = np.zeros((len(self.M()), 1))
+
+        # place gravity effect on disk nodes
```

Comment:
I think we should find a way to make this part more readable with something like:
```python
for disk in self.disks:
    grav[4 * disk.node_number - 3] = -9.8065
```
This is just an idea...